middleman.js

```diff
@@ -678,7 +678,8 @@ const render = (content, options = {}) => {
           }
 
           distilled = await autofill(page, match.distilled);
-          await autoclick(page, distilled);
+          await clicks(page, distilled, '[gg-autoclick]');
+          await clicks(page, distilled, '[type="submit"]');
```

Comment:
We need to facilitate gradual migration so the first selector should be `[gg-autoclick]:not(button)` and the second one can be `button[gg-autoclick], button[type=submit]`. That way, all the specs do not need to switch to `type=submit` yet.